plaid/plaid-link-android
# Make sample code easier to read

Made many minor improvements:

- Update dependencies to latest (stable) version
- Remove unneeded files
- Simplify theming
- Update color scheme
- Simplify layouts
- Add java suffix to all java files, kotlin files don't have a suffix
- Use a Plaid logo
- Rename app to Link demo, so the title is displayed fully in a launcher
- Code cleanup and simplification

<img src="https://user-images.githubusercontent.com/6319637/78637525-d023c500-78aa-11ea-9989-c2bf300dadd2.png" width="260">&emsp;<img src="https://user-images.githubusercontent.com/6319637/78637536-d44fe280-78aa-11ea-93a1-aee64c036402.png" width="260">
